tests/sanic/app.py

```diff
@@ -0,0 +1,58 @@
+from urllib.parse import urlencode
+
+import pytest
+from sanic import Sanic
+from sanic.testing import SanicTestClient
+
+from graphql_server.sanic import GraphQLView
+
+from .schema import Schema
+
+
+def create_app(path="/graphql", **kwargs):
+    app = Sanic(__name__)
+    app.debug = True
+
+    schema = kwargs.pop("schema", None) or Schema
+    # async_executor = kwargs.pop("async_executor", False)
+    #
+    # if async_executor:
+    #
+    #     @app.listener("before_server_start")
+    #     def init_async_executor(app, loop):
+    #         executor = AsyncioExecutor(loop)
+    #         app.add_route(
+    #             GraphQLView.as_view(schema=schema, executor=executor, **kwargs), path
+    #         )
+    #
+    #     @app.listener("before_server_stop")
+    #     def remove_graphql_endpoint(app, loop):
+    #         app.remove_route(path)
+    #
+    # else:
+    #     app.add_route(GraphQLView.as_view(schema=schema, **kwargs), path)
```

Comment:
Remove this?

Reply:
Yeah, those code blocks related to executor setup should be removed but I always worry about it as executor option is still supported on these server implementations. Take as example [flask-graphql](https://github.com/graphql-python/graphql-server-core/blob/master/graphql_server/flask/graphqlview.py#L22), which has the `executor` parameter to be an extra option passed to **run_http_query** function and do something with it.

I assume these tests are handled by the internal `graphql-core` library.

Reply:
@KingDarBoja as far as I know the executor support was just to allow support for asyncio and since it's not supported natively it's not longer needed. I've created a PR to remove it from the Flask code actually: #40